packages/@aws-cdk/aws-stepfunctions-tasks/test/ecs/run-tasks.test.ts

```diff
@@ -152,6 +152,7 @@ test('Running a Fargate Task', () => {
     launchTarget: new tasks.EcsFargateLaunchTarget({
       platformVersion: ecs.FargatePlatformVersion.VERSION1_4,
     }),
+    propagateTags: ecs.PropagatedTagSource.SERVICE,
```

Comment:
I'd like to see a test for each of the allowed values and one where this prop is not provided.